lib/keycloak-admin/client/client.rb

```diff
@@ -38,19 +34,13 @@ def created_id(response)
     end
 
     def create_payload(value)
-      if value.nil?
-        ""
-      elsif value.kind_of?(Array)
-        "[#{value.map(&:to_json) * ","}]"
+      return "" if value.nil?
```

Comment:
It looks like this code has been refactored automatically by your editor. This style is not consistent with the rest of the project. This should not be part of the PR.